### PR TITLE
Ps1 cleanup

### PR DESCRIPTION
**Pull request by ishanpan**
Edited files according to specification

### CODE REVIEW

File: awless/install.ps1 [hunk 1]
```diff
@@ -44,14 +44,14 @@ IF (!(Test-Path -Path "$pkg_src_cmd")) {
 
     # Settle unpacked archive into place
     echo "Install Location: $pkg_src_cmd"
-    New-Item "$pkg_src_bin" -ItemType Directory -Force
```
Comment (coolaj86) [inline awless/install.ps1:47]: this should not be moved, only the instances that are directly nested in an if statement should be moved.

File: vim-ale/install.ps1 [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env pwsh
 
 IF (!(Test-Path -Path "$Env:USERPROFILE\.vim\pack\plugins\start")) {
```
Comment (coolaj86) [inline vim-ale/install.ps1:3]: this is the sort of IF block that should be removed

**Comment by coolaj86** [comment]
Hey @ishanpan, this is not quite right. Please take a more careful look at the description.

I'd recommend doing this in three commits:
1. use an automated tool to add ` | out-null` to the end of every `Remove-Item`, commit
2. use an automated tool to add ` | out-null` to the end of every `New-Item`, commit
3. manually (or automatically) remove all `IF ... Test-Path` blocks, but leave the `New-Item` line in-place (minus the leading whitespace), commit

**Comment by ishanpan** [comment]
> Hey @ishanpan, this is not quite right. Please take a more careful look at the description.
> 
> I'd recommend doing this in three commits:
> 
> 1. use an automated tool to add ` | out-null` to the end of every `Remove-Item`, commit
> 2. use an automated tool to add ` | out-null` to the end of every `New-Item`, commit
> 3. manually (or automatically) remove all `IF ... Test-Path` blocks, but leave the `New-Item` line in-place (minus the leading whitespace), commit

Thank you for the feedback. I will rectify my mistake and get back to you.

**Pull request closed without merging**